src/test/java/uk/sky/cqlmigrate/SchemaUpdatesTest.java

```diff
@@ -119,19 +106,20 @@ public void rowInsertedWithMessageDigestHashingAlgorithmIsSameAsGuavaSha1Hashing
 
         //when
         schemaUpdates.add(
-            filename,
-            Paths.get(cqlResource.toURI())
+                filename,
+                Paths.get(cqlResource.toURI())
         );
 
         //then
         final String guavaSha1Hash = Resources.asByteSource(cqlResource).hash(Hashing.sha1()).toString();
         final ResultSet resultSet = session.execute("SELECT * from " + SCHEMA_UPDATES_TABLE);
-        assertThat(resultSet.all())
-            .hasOnlyOneElementSatisfying(row -> {
-                assertThat(row.getString("filename"))
+        assertThat(resultSet.all().size()).isEqualTo(1);
+        resultSet.forEach(row ->
+        {
+            assertThat(row.getString("filename"))
                     .isEqualTo(filename);
-                assertThat(row.getString("checksum"))
+            assertThat(row.getString("checksum"))
                     .isEqualTo(guavaSha1Hash);
-            });
+        });
     }
```

Comment:
any reason to change this out of curiosity? Don't mind either way.

Reply:
cant remember why. i ll change it back